vg/vggio/vggio_test.go

```diff
@@ -24,6 +26,45 @@ import (
 
 const deltaGio = 0.05 // empirical value from experimentation.
 
+func init() {
```

Comment:
A broad overview comment for this init would be helpful.

Reply:
let's address this part on https://github.com/gonum/plot/pull/718